components/safety_board/HW/include/SystemConfig.h

```diff
@@ -0,0 +1,86 @@
+/**
+ * @file SystemConfig.h
+ * @brief  Header file for the system configuration
+ * @author Joshua Lafleur (josh.lafleur@outlook.com)
+ * @version 0.1
+ * @date 2023-01-13
+ */
+
+#pragma once
+
+/******************************************************************************
+ *                             I N C L U D E S
+ ******************************************************************************/
+
+#include "stm32f1xx.h"
+
+#include "ErrorHandler.h"
+
+
+/******************************************************************************
+ *                              D E F I N E S
+ ******************************************************************************/
+
+#define BMS_STATUS_ID 0x037
```

Comment:
these being hard-coded is fine for now, but this should come from the network generator tool eventually, and you should add the definition for these messages to it in this PR as well (as a source of truth that is human readable)